docs/custom.md

```diff
@@ -151,65 +150,53 @@ You can now create analogous files `create_box.archive.yaml`, `insert_water.arch
+      Files:
+      - file: 'Custom_ELN_Entries/water.gro'
+        description: 'The structure file for simulation input.'
     ```
 
 ??? success "`workflow_parameters.archive.yaml`"
 
     ```yaml
-        data:
-          m_def: nomad.datamodel.metainfo.eln.ElnBaseSection
-          name: 'Workflow Parameters'
-          description: 'This is a description of the overall workflow parameters, or alternatively standard workflow specification...'
+    data:
+      m_def: nomad.datamodel.metainfo.eln.ElnBaseSection
+      name: 'Workflow Parameters'
+      description: 'This is a description of the overall workflow parameters, or alternatively standard workflow specification...'
     ```
 
 ??? success "`workflow_scripts.archive.yaml`"
 
     ```yaml
-        data:
-          m_def: '../upload/raw/Custom_ELN_Entries/ELNFiles.archive.yaml#ELNAnnotatedFiles'
-          name: 'Workflow Scripts'
-          description: 'All the scripts run during setup of the MD simulation.'
-          Files:
-          - file: 'Custom_ELN_Entries/workflow_script_1.py'
-              description: 'Creates the simulation box and inserts water molecules.'
-          - file: 'Custom_ELN_Entries/workflow_script_2.py'
-              description: 'Creates the appropriate force field files for the simulation engine.'
+    data:
+      m_def: '../upload/raw/Custom_ELN_Entries/ELNFiles.archive.yaml#ELNAnnotatedFiles'
+      name: 'Workflow Scripts'
+      description: 'All the scripts run during setup of the MD simulation.'
+      Files:
+      - file: 'Custom_ELN_Entries/workflow_script_1.py'
+        description: 'Creates the simulation box and inserts water molecules.'
+      - file: 'Custom_ELN_Entries/workflow_script_2.py'
+        description: 'Creates the appropriate force field files for the simulation engine.'
     ```
 
 ## Creating a custom workflow in NOMAD
 
-NOMAD allows users to connect entries into a workflow, i.e., a directed graph structure. This is achieved using the same parsing functionality as demonstrated with the custom schemas above. In this case, we simply populate the `workflow2` section instead of the `data` section. When uploaded to NOMAD, a new _workflow_ entry will be created, with references to each of the workflow tasks, and also an interactive workflow graph for easy navigation of the entire workflow.
+NOMAD allows users to connect entries into a workflow, i.e., a directed graph structure. This is achieved using the same parsing functionality as demonstrated with the custom schemas above. In this case, we simply populate the `workflow2` section instead of the `data` section. Learn more on the [archive file structure](https://nomad-lab.eu/prod/v1/test/docs/explanation/data.html#archive-files-a-shared-entry-structure) from official NOMAD documentation. When uploaded to NOMAD, a new _workflow_ entry will be created, with references to each of the workflow tasks, and also an interactive workflow graph for easy navigation of the entire workflow.
```

Comment:
"Learn more **about** the archive file structure **in** the official NOMAD documentation."

I'd put the link on "official NOMAD documentation", but that's less pressient.

Reply:
I think your addition is good, but it does break up the thought flow. Maybe at the end of the paragraph?

Reply:
👍🏼